tests/utils.py

```diff
@@ -36,13 +37,11 @@ def iterate_schema(schema: dict) -> Generator[Tuple[Optional[dict], Optional[Res
                         continue
                     schema_section = None
                     response = None
-                    try:
-                        if "content" in responses_object.keys():
+                    with suppress(KeyError):
```

Comment:
Instead of 
```python
try: 
    something
except ValueError:
    pass
```

You can do this 🥳
```python
with suppress(ValueError):
    something
```  

https://docs.python.org/3/library/contextlib.html#contextlib.suppress